packages/ui-app/src/Collection.tsx

```diff
@@ -53,9 +53,9 @@ export default class Collection<P extends CollectionProps, S extends CollectionS
     return !children || (Array.isArray(children) && children.length === 0);
   }
 
-  public static getDerivedStateFromProps ({ children }: CollectionProps): Pick<any, never> {
+  public static getDerivedStateFromProps ({ isEmpty, children }: CollectionProps): CollectionState {
```

Comment:
what exactly is `Pick<any, never>`?

Reply:
It should be `Pick<State, never>` - which is not pretty, but matches up with the actual underlying React definitions. (`Partial<State>` does not work in these cases where you only return some of the keys)